src/_includes/our_work_org_card.liquid

```diff
@@ -0,0 +1,35 @@
+<section class="hero org-card section--bg-linen">
+    <div class="container">
+        <div class="col-12">
+            <div class="content--wrapper">
+                <div class="tablet-hide desktop-hide" style="margin-top: 120px;">
+                    <img class="org-logo" src="{{ logo }}" />
+                </div>
+                <div class="section--intro">
+                    <div class="col-6 col-12-mobile">
+                        <h1>{{ name }}</h1>
+                    </div>
+                    <div class="col-6 col-12-mobile content--right display-flex mobile-hide">
+                        <img class="org-logo" src="{{ logo }}" />
+                    </div>
+                </div>
+                <div class="section--intro">
+                    <div class="col-6 col-12-tablet col-12-mobile">
+                        <p class="section--subtitle">{{ type_of_work }}</p>
+                        <div class="pad-right">
```

Comment:
if we need true utility classes, we should add them to `_foundation`, otherwise we should just make a class for this section within `org-card`, ie,

```
.org-card {
  &__content {}
}
```

or something semantically similar.

Reply:
This is pretty specific to `/our-work/` so I opted for the latter!